clean_project/project/search_engine.py

```diff
@@ -0,0 +1,31 @@
+from index import load_config_file
+import psycopg2
+
+
+class SearchEngine(object):
+    def __init__(self, connect_path):
+        self.connect_path = connect_path
+
+    def search(self, search_word):
+        """
+        Search if a given word is in bucket of files
+        :param search_word: Word given to search
+        :return: List of tuples composed by the given word, file and all the occurrences found
+        """
+        result_search = []
+        database = load_config_file(self.connect_path).get('database')
```

Comment:
instead of parsing these properties every time this method is called, you should do it only once in the initializer. 

also, you should factor out the load_config_file() call because in this way you're opening the file *several* time, with a big impact on performance. as an exercise, time how long the call takes now and how long it takes with the suggestion provided

also, why does this module depend on index? load_config_file, if it's a common method, should really be in a common module, not in index